### DISCUSSION TITLE
Importing Attribute Values from List

### DISCUSSION THREAD

**Discussion by paultvis** [atrocore/atrocore#763] [General]
Hi,

Following on from my last discussion on the new List approach, we have just tried running an import to add attribute values in a list to products. This has been done previously, before the lists were moved out but two issues were encountered:
1. The import could only be run using create for the product_attribute_value. If we had had the attributes already assigned to the product but empty, update was not able to update the product_attribute_value
2. On adding the values (once the product_attribute_values were removed - another fun task as that cannot be done through bulk update and had to be done directly in the db), the product_attribute_value was created, but the link through to the extensible_enum and extensible_enum_option was not created. So the result was that the attribute was added to the product, but the value given for the attribute was not imported.

So, the first question is - are we doing the import wrong? And if not, will this issue get looked at?

Thanks,

Paul

**Comment by rratsun** [comment]
For solving your problem, you need to have two import feeds:
* Import ExtensibleEnumOption. Here you need to create options for your list but also you have field code. Field code you need for defining your option.
* Import ProductAttributeValue. Here into field 'value' you can set code of option and that's all. System will automatically define option by code.
